eo2js/src/commands/transpile.js

```diff
@@ -82,7 +145,7 @@ const needsRetranspile = function(source, transpiled) {
 const transform = function(tojo, options, transformations, parser) {
   const text = fs.readFileSync(tojo[verified]).toString()
   let xml = parser.parse(text)
-  const pth = pathFromName(xml.program['@_name'])
+  const pth = pathFromName(getProgramName(xml))
```

Comment:
@konard to get `pth` you need to take top level `<o>` name:
```xml
<object>
  <o name="foo"/>
</object>
```

For instance, for such xml `pth` is foo